packages/playground/windows/playground/MainPage.cpp

```diff
@@ -167,6 +172,18 @@ void winrt::playground::implementation::MainPage::x_UseWebDebuggerCheckBox_Unche
   }
 }
 
+void winrt::playground::implementation::MainPage::x_Theme_SelectionChanged(
+    winrt::Windows::Foundation::IInspectable const &sender,
+    winrt::Windows::UI::Xaml::Controls::SelectionChangedEventArgs const &e) {
+  auto theme = ElementTheme::Default;
+  if (x_Theme().SelectedItem() == x_themeLight()) {
+    theme = ElementTheme::Light;
+  } else if (x_Theme().SelectedItem() == x_themeDark()) {
+    theme = ElementTheme::Dark;
+  }
+  xaml::Window::Current().Content().as<xaml::FrameworkElement>().RequestedTheme(theme);
```

Comment:
We should probably add logic to the playground-win32 app as well, and make it window message based since CoreWindow likely won't work in xaml islands in winui3.

Reply:
Yup, I'll address playground-win32 in separate PR.